packages/admin/admin-date-time/src/dateTimePicker/DateTimePicker.tsx

```diff
@@ -9,7 +9,7 @@ import { TimePicker as TimePickerBase } from "../timePicker/TimePicker";
 import { getIsoDateString } from "../utils/datePickerHelpers";
 import { getDateWithNewTime, getTimeStringFromDate } from "../utils/timePickerHelpers";
 
-export type DateTimePickerClassKey = "root" | "dateFormControl" | "timeFormControl" | "datePicker" | "timePicker";
+type DateTimePickerClassKey = "root" | "dateFormControl" | "timeFormControl" | "datePicker" | "timePicker";
```

Comment:
@jamesricky is it okay to remove this export? Does this cause problems with theming?